mass_mailing_welcome_mail/data/base_automation_data.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <data>
+        <record id="email_template" model="mail.template">
+            <field name="name">Welcome mail - Mailing List</field>
+            <field name="model_id" ref="mass_mailing.model_mail_mass_mailing_contact"/>
+            <field name="use_default_to" eval="True"/>
+            <field name="email_from">${object.company_id.email}</field>
+            <field name="lang">${object.env.context.get("lang")}</field>
+            <field name="subject">Welcome to the mailing list</field>
+            <field name="body_html" type="html">
+                <div>Dear <b>${object.name}</b>,
+                    <br/>
+                    <br/>
+                    <h3>Thanks for signing up four our newsletter!</h3>
+                    <p>
+                        Enjoy a discount code for 20% off your first order
```

Comment:
This shouldn't be included here in this mail, but customize the template on the customer.

Reply:
This is part of the default template of the newsletter in Odoo, something very common, that's why I included it here. I delete it.